### ISSUE TITLE
Incompatible with thor 0.19.2 dependency

### ISSUE THREAD

**Issue by kenaniah** [ddollar/foreman#653]
After upgrading to `thor-0.19.2` from `thor-0.19.1`, the following issue is takes place when running `foreman`:
```
/.../lib/ruby/gems/2.3.0/gems/thor-0.19.2/lib/thor/base.rb:534:in `thor_reserved_word?': "run" is a Thor reserved word and cannot be defined as command (RuntimeError)
	from /.../lib/ruby/gems/2.3.0/gems/thor-0.19.2/lib/thor/base.rb:597:in `method_added'
	from /.../lib/ruby/gems/2.3.0/gems/foreman-0.82.0/lib/foreman/cli.rb:80:in `<class:CLI>'
	from /.../lib/ruby/gems/2.3.0/gems/foreman-0.82.0/lib/foreman/cli.rb:11:in `<top (required)>'
	from /.../lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /.../lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
	from /.../lib/ruby/gems/2.3.0/gems/foreman-0.82.0/bin/foreman:5:in `<top (required)>'
	from /.../bin/foreman:23:in `load'
	from /.../bin/foreman:23:in `<main>'
```

Everything works perfectly with `thor-0.19.1` and below.

**Comment by stevenleeg** [comment]
I've been seeing this issue as well with a fresh `rails init`ed project

Here's my Procfile:

```
web: bundle exec rails s
```

Running `foreman start` gives the same error as above.

**Comment by MrChrisW** [comment]
This is fixed in https://github.com/erikhuda/thor/commit/329cff6692c040fec7dfb67cc1fa742d31d5b97d and released in Thor `0.19.4`

**Comment by kenaniah** [comment]
Indeed, fixed upstream.
